SYCL/CommandGraph/graph_common.hpp

```diff
@@ -0,0 +1,131 @@
+// Common header for SYCL command_graph tests
+
+#include <sycl/sycl.hpp>
+
+#include <sycl/ext/oneapi/experimental/graph.hpp>
+
+#include <numeric>
+
+// Some test constants
+constexpr size_t size = 1024;
+constexpr unsigned iterations = 5;
+
+// Kernel declarations for use in run_kernels()
+class increment_kernel;
+class add_kernel;
+class subtract_kernel;
+class decrement_kernel;
+
+// Kernel declarations for use in run_kernels_usm()
+class increment_kernel_usm;
+class add_kernel_usm;
+class subtract_kernel_usm;
+class decrement_kernel_usm;
+
+using namespace sycl;
+
+/** Runs a series of kernels based on the examples from the initial Graphs
+ * sample code repo.
+ *
+ */
+template <typename T>
+void run_kernels(sycl::queue q, const size_t size, buffer<T> dataA,
+                 buffer<T> dataB, buffer<T> dataC) {
+  // Read & write Buffer A
+  q.submit([&](handler &cgh) {
+    auto pData = dataA.template get_access<access::mode::read_write>(cgh);
```

Comment:
did you try removing the `.template`? Thought this might have been an artefact of our old setup

Reply:
I've removed `.template` from most instances but the uses within the `run_kernels` functions in the common header still require it.